### PR TITLE
[BEAM-1661] shade guava in beam-sdks-java-io-jdbc

### PR DESCRIPTION
**Pull request by XuMingmin**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by davorbonaci** [comment]
R: @aviemzur 
CC: @ssisk @jbonofre @davorbonaci

**Comment by aviemzur** [comment]
Please `make sure the PR title is formatted like:
[BEAM-<Jira issue #>] Description of pull request`

File: sdks/java/io/jdbc/pom.xml [hunk 1]
```diff
@@ -166,4 +166,66 @@
...
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <systemPropertyVariables>
+            <beamUseDummyRunner>false</beamUseDummyRunner>
+          </systemPropertyVariables>
+        </configuration>
+      </plugin>
```
Comment (aviemzur) [inline sdks/java/io/jdbc/pom.xml]: Why is this surefire configuration needed here?
The tests seem to run fine without this.

File: sdks/java/io/jdbc/pom.xml [hunk 2]
```diff
@@ -166,4 +166,54 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <!-- BEAM-1661 -->
+        <plugin>
```
Comment (aviemzur) [inline sdks/java/io/jdbc/pom.xml]: This findbugs plugin node can be removed.

**Comment by aviemzur** [comment]
Retest this please.

**Comment by XuMingmin** [comment]
The failures are related with Metrics, junit test for JdbcIO are good. https://builds.apache.org/job/beam_PreCommit_Java_MavenInstall/8275/org.apache.beam$beam-sdks-java-io-jdbc/

**Comment by jbonofre** [comment]
Reviewing and merging (when green).

**Pull request closed without merging**